tests/apollo/util/bft.py

```diff
@@ -898,3 +899,28 @@ async def num_of_slow_path(self):
                         except KeyError:
                             # metrics not yet available, continue looping
                             pass
+    
+    async def do_key_exchange(self):
+        self.start_all_replicas()
+        with trio.fail_after(seconds=20):
```

Comment:
as demonstrated by @oded282, let's wrap this as an Eliot action for better logging